### PR TITLE
Update fusionx details info

### PR DESCRIPTION
**Pull request by NIXBNT**
Successful Tenderly test tx w splitter

### CODE REVIEW

File: fastlane_bot/config/network.py [hunk 1]
```diff
@@ -237,7 +237,7 @@ class ConfigNetwork(ConfigBase):
 
     GAS_ORACLE_ADDRESS = None
 
-    CARBON_V1_FORKS = [CARBON_V1_NAME, CARBON_V1_X2_NAME]
+    CARBON_V1_FORKS = [CARBON_V1_NAME, CARBON_V1_FORK1_NAME]
```
Comment (Lesigh-3100) [inline fastlane_bot/config/network.py:240]: I think this can be removed

File: fastlane_bot/config/constants.py [hunk 2]
```diff
@@ -20,6 +20,6 @@
 FUSIONX_V3_NAME = "fusionx_v3"
 CLEOPATRA_V3_NAME = "cleopatra_v3"
 CARBON_V1_NAME = "carbon_v1"
-CARBON_V1_FORK1_NAME = "altered_carbon"
+CARBON_V1_FORK1_NAME = "fusionx_carbon"
```
Comment (Lesigh-3100) [inline fastlane_bot/config/constants.py:23]: I believe this can be removed

File: fastlane_bot/config/network.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 
 from . import selectors as S
 from .base import ConfigBase
-from .constants import CARBON_V1_FORK1_NAME, CARBON_V1_NAME
+from .constants import CARBON_V1_NAME
```
Comment (barakman) [inline fastlane_bot/config/network.py:15]: Not clear why this is needed here (no other change in this PR seems to be using it)

File: fastlane_bot/config/network.py [hunk 4]
```diff
@@ -237,7 +236,7 @@ class ConfigNetwork(ConfigBase):
 
     GAS_ORACLE_ADDRESS = None
 
-    CARBON_V1_FORKS = [CARBON_V1_NAME, CARBON_V1_X2_NAME]
+    CARBON_V1_FORKS = []
```
Comment (barakman) [inline fastlane_bot/config/network.py:239]: Not clear why this is needed here (why not just create it in the class's post-init function?)

**Review by barakman — CHANGES_REQUESTED** [review]

**Review by barakman — APPROVED** [review]

**Pull request merged**